[2025-08-01T13:33:26.082+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_medallon.ejecutar_merge manual__2025-08-01T13:32:38.068363+00:00 [queued]>
[2025-08-01T13:33:26.207+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_medallon.ejecutar_merge manual__2025-08-01T13:32:38.068363+00:00 [queued]>
[2025-08-01T13:33:26.208+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-01T13:33:26.295+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): ejecutar_merge> on 2025-08-01 13:32:38.068363+00:00
[2025-08-01T13:33:26.300+0000] {standard_task_runner.py:60} INFO - Started process 215 to run task
[2025-08-01T13:33:26.304+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_medallon', 'ejecutar_merge', 'manual__2025-08-01T13:32:38.068363+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_medallon_dag.py', '--cfg-path', '/tmp/tmpgx8dhwgm']
[2025-08-01T13:33:26.307+0000] {standard_task_runner.py:88} INFO - Job 120: Subtask ejecutar_merge
[2025-08-01T13:33:26.457+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_medallon.ejecutar_merge manual__2025-08-01T13:32:38.068363+00:00 [running]> on host 70eaa744a05c
[2025-08-01T13:33:26.669+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo.biasotto' AIRFLOW_CTX_DAG_ID='pipeline_medallon' AIRFLOW_CTX_TASK_ID='ejecutar_merge' AIRFLOW_CTX_EXECUTION_DATE='2025-08-01T13:32:38.068363+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-01T13:32:38.068363+00:00'
[2025-08-01T13:33:26.670+0000] {sql.py:276} INFO - Executing: CALL merge_bulk_to_final();
[2025-08-01T13:33:26.685+0000] {base.py:83} INFO - Using connection ID 'postgres_medallon' for task execution.
[2025-08-01T13:33:26.801+0000] {base.py:83} INFO - Using connection ID 'postgres_medallon' for task execution.
[2025-08-01T13:33:26.836+0000] {sql.py:450} INFO - Running statement: CALL merge_bulk_to_final();, parameters: None
[2025-08-01T13:33:27.186+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=pipeline_medallon, task_id=ejecutar_merge, execution_date=20250801T133238, start_date=20250801T133326, end_date=20250801T133327
[2025-08-01T13:33:27.241+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-01T13:33:27.270+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
